src/chrome/content/rules/carnegiehub.org.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+	Invalid certificate:
+		fax.carnegiehub.org
+		files.carnegiehub.org
+
+	Refused:
+		carnegiehub.org
+		www.carnegiehub.org
+		calendar.carnegiehub.org
+		docs.carnegiehub.org
+		fsp.carnegiehub.org
+		groups.carnegiehub.org
+		intranet.carnegiehub.org
+		isp.carnegiehub.org
+		mail.carnegiehub.org
+-->
+<ruleset name="carnegiehub.org (partial)">
+	<target host="filedrop.carnegiehub.org" />
```

Comment:
This host hit an error:
`ERROR src/chrome/content/rules/carnegiehub.org.xml: Fetch error: http://filedrop.carnegiehub.org/ => https://filedrop.carnegiehub.org/: (60, 'SSL certificate problem: unable to get local issuer certificate')`